cam/cam.go

```diff
@@ -337,10 +339,10 @@ func (vs *videostore) Properties(_ context.Context) (camera.Properties, error) {
 	return camera.Properties{}, nil
 }
 
-// processFrames reads frames from the camera, encodes, and writes to the segmenter
-// which chunks video stream into clip files inside the storage directory.
-// TODO(seanp): Should this be throttled to a certain FPS?
-func (vs *videostore) processFrames(ctx context.Context) {
+// fetchFrames reads frames from the camera and stores the decoded image
+// in the latestFrame atomic pointer. This routine runs as fast as possible
+// to keep the latest frame up to date.
+func (vs *videostore) fetchFrames(ctx context.Context) {
```

Comment:
We may want a benchmark tests for this, either now or a future pr.

Reply:
sounds good -- have performance ticket up next so include benchmarks in that diff.